x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -106,22 +108,37 @@ describe('Case View Page activity tab', () => {
   });
   let appMockRender: AppMockRenderer;
 
+  const platinumLicense = licensingMock.createLicense({
+    license: { type: 'platinum' },
+  });
+
+  const basicLicense = licensingMock.createLicense({
+    license: { type: 'basic' },
+  });
+
   beforeEach(() => {
     appMockRender = createAppMockRenderer();
   });
 
-  it('should render the activity content and main components', () => {
+  it('should render the activity content and main components', async () => {
+    appMockRender = createAppMockRenderer({ license: platinumLicense });
     const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
+
     expect(result.getByTestId('case-view-activity')).toBeTruthy();
     expect(result.getByTestId('user-actions')).toBeTruthy();
     expect(result.getByTestId('case-tags')).toBeTruthy();
     expect(result.getByTestId('connector-edit-header')).toBeTruthy();
     expect(result.getByTestId('case-view-status-action-button')).toBeTruthy();
     expect(useGetCaseUserActionsMock).toHaveBeenCalledWith(caseData.id, caseData.connector.id);
+
+    await waitForComponentToUpdate();
```

Comment:
Depending on the component, it is possible that after the end of the test a component will do an update to a state. This will generate the "not wrapped in act(...)" warning. This piece of code waits for all updates of the component before the end of the test.

You can read more about it here https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning